controllers/topicPage/TopicPage.js

```diff
@@ -39,7 +41,13 @@ class TopicPage {
       payload.listPostAnonymous
     ).getHasBlock(topicPages);
     const threshold = ACTIVITY_THRESHOLD.TOPIC_FEED;
-    results.data = await filterFeeds(payload.userId, newTopicPagesWithBlock, payload.id, threshold);
+    results.data = await filterFeeds(
+      payload.userId,
+      payload.anonymousUserId,
+      newTopicPagesWithBlock,
+      payload.id,
+      threshold
+    );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `filterFeeds` function now accepts an additional parameter `anonymousUserId`. Ensure that this change is reflected wherever the function is called. Also, make sure to handle cases where `anonymousUserId` might be `null` or `undefined` in the `filterFeeds` function.
```diff
-    results.data = await filterFeeds(payload.userId, newTopicPagesWithBlock, payload.id, threshold);
+    results.data = await filterFeeds(
+      payload.userId,
+      payload.anonymousUserId,
+      newTopicPagesWithBlock,
+      payload.id,
+      threshold
+    );
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->